modules/ROOT/examples/live-demos/full-featured/index.js

```diff
@@ -583,10 +1006,20 @@ tinymce.ScriptLoader.loadScripts(['https://cdn.jsdelivr.net/npm/faker@5/dist/fak
       }
     },
     revisionhistory_fetch: fetchRevisions,
-    revisionhistory_author: {
-      id: 'john.doe',
-      name: 'John Doe'
-    },
+    // revisionhistory_author: {
+    //   id: 'john.doe',
+    //   name: 'John Doe'
+    // },
```

Comment:
@MitchC1999 please advise if this can be removed- providing the implementation is correct for using the `user_id` 
```suggestion
```